packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx

```diff
@@ -2,117 +2,180 @@ import { Meta, Story, ArgsTable, Canvas } from '@storybook/addon-docs/blocks';
 import ScalePagination from './ScalePagination.vue';
 
 <Meta
-    title="Beta Components/Pagination"
-    component={ScalePagination}
-    argTypes={{
-        hideBorders: {
-            table: {
-              type: { summary: 'Boolean' },
-              defaultValue: { summary: false },
-            },
-            description: `(optional) Set to true to hide top and bottom borders`,
-            control: { type: null }
-        },
-        pageSize: {
-            table: {
-              type: { summary: 'number' },
-              defaultValue: { summary: 10 },
-            },
-            description: `(optional) Set number of rows/elements to show per page`,
-            control: { type: null }
-        },
-        startElement: {
-            table: {
-              type: { summary: 'number' },
-              defaultValue: { summary: 0 },
-            },
-            description: `(optional) Index of first element to display`,
-            control: { type: null }
-        },
-        totalElements: {
-            table: {
-              type: { summary: 'number' },
-              defaultValue: { summary: 1 },
-            },
-            description: `(optional) Total number of rows/elements used to calculate page displays`,
-            control: { type: null }
-        },
-        styles: {
-            table: {
-              type: { summary: 'string' },
-            },
-            description: `(optional) Injected styles`,
-            control: { type: null }
-        },
-    }}
+  title="Beta Components/Pagination"
+  component={ScalePagination}
+  argTypes={{
+    hideBorders: {
+      table: {
+        type: { summary: 'Boolean' },
+        defaultValue: { summary: false },
+      },
+      description: `(optional) Set to true to hide top and bottom borders`,
```

Comment:
Thank you for taking the time for adding these descriptions 🙏

Reply:
...like this one, too :-)